SDKTest/EventTests.swift

```diff
@@ -353,4 +361,90 @@ class EventTests: XCTestCase {
         }
         assert(neuroHTTPRequest != nil)
     }
+    
+    // Specific UI Class Registration Tests
+    func testUITextFieldRegistration() {
+        NeuroID.start()
+        let testView = UITextField()
+        testView.id = "UITextField"
+        NeuroID.manuallyRegisterTarget(view: testView)
+        let events = DataStore.getAllEvents()
+        let filteredRegisteredTargets = events.filter { $0.type == "REGISTER_TARGET" }
+        assert(filteredRegisteredTargets.count == 1)
+        assert(filteredRegisteredTargets[0].et == "UITextField::UITextField")
+    }
+    
+    func testUITextViewRegistration() {
+        NeuroID.start()
+        let testView = UITextView()
+        testView.id = "UITextView"
+        NeuroID.manuallyRegisterTarget(view: testView)
+        let events = DataStore.getAllEvents()
+        let filteredRegisteredTargets = events.filter { $0.type == "REGISTER_TARGET" }
+        assert(filteredRegisteredTargets.count == 1)
+        assert(filteredRegisteredTargets[0].et == "UITextView::UITextView")
+    }
+    
+    func testUIButtonRegistration() {
+        NeuroID.start()
+        let testView = UIButton()
+        testView.id = "UIButton"
+        NeuroID.manuallyRegisterTarget(view: testView)
+        let events = DataStore.getAllEvents()
+        let filteredRegisteredTargets = events.filter { $0.type == "REGISTER_TARGET" }
+        assert(filteredRegisteredTargets.count == 1)
+        assert(filteredRegisteredTargets[0].et == "UIButton::UIButton")
+    }
+    
+    func testUIDatePickerRegistration() {
+        NeuroID.start()
+        let testView = UIDatePicker()
+        testView.id = "UIDatePicker"
+        NeuroID.manuallyRegisterTarget(view: testView)
+        let events = DataStore.getAllEvents()
+        let filteredRegisteredTargets = events.filter { $0.type == "REGISTER_TARGET" }
+        assert(filteredRegisteredTargets.count == 1)
+        assert(filteredRegisteredTargets[0].et == "UIDatePicker::UIDatePicker")
+    }
+    
+    // UI Class Registrations that are NOT implemented
+    func testUISliderNotRegistered() {
```

Comment:
Again might be overkill but these tests make sure that we can't capture the other UI elements we haven't prepped for